gui/docks/labeldock.cpp

```diff
@@ -54,13 +54,15 @@ void LabelDock::init()
 	ui->labelView->setDragDropMode(QAbstractItemView::NoDragDrop);
 	ui->labelView->setSpacing(0);
 	ui->labelView->setUniformItemSizes(true);
+    ui->labelView->setSelectionMode(QAbstractItemView::MultiSelection);
```

Comment:
Can this and the above properties also be set in the .ui file? (using Designer or QtCreator's Designer module)